tokens/manage/multisig-safe.md

```diff
@@ -312,4 +313,36 @@ To double check that the correct text was set, you can go through the process ag
 
 Congratulations, you've successfully interacted with a smart contract using the Safe!
 
+## Using Moonbeam Safe APIs
+There are APIs available to read from and interact with Moonbeam Safes for Moonbase Alpha, Moonriver, and Moonbeam.
+
+|     Network    |                                           Multisig API Page                                              |
+|:--------------:|:--------------------------------------------------------------------------------------------------------:|
+|    Moonbeam    |  [{{ networks.moonbeam.multisig.api_page }}]({{ networks.moonbeam.multisig.api_page }}){target=_blank}   |
+|    Moonriver   | [{{ networks.moonriver.multisig.api_page }}]({{ networks.moonriver.multisig.api_page }}){target=_blank}  |
+| Moonbase Alpha |  [{{ networks.moonbase.multisig.api_page }}]({{ networks.moonbase.multisig.api_page }}){target=_blank}   |
```

Comment:
for something like this i would typically just use tabs since it's for each network, so it would be something like this: https://github.com/PureStake/moonbeam-docs/blame/master/builders/build/canonical-contracts/precompiles/democracy.md#L17-L30